### PR TITLE
chore: automate beta and ga creation from CI COMPASS-6270

### PR DESCRIPTION
**Pull request by mcasimir**
- Switch the branching model to 2 branches: `beta-releases` and `ga-releases`.
- Creates a new beta at 5 AM UTC on Monday merging `main` to `beta-releases`, can be scheduled manually. Uses the latest know release in Jira to bump the version. 
- Adds a workflow to promote the latest beta to a GA (merge a beta branch). Uses the `fixVersion` of a release ticket passed as input to bump the version.

Covers quickfixes and releasing GAs from alternate branches through workflow inputs.

### CODE REVIEW

File: packages/notary-service-client/README.md [hunk 1]
```diff
@@ -4,52 +4,6 @@
 
 ## Example
 
-## CLI
```
Comment (mcasimir) [inline packages/notary-service-client/README.md:7]: The notary service is going to be decommissioned in the upcoming months, didn't see a good reason to keep this here

File: .github/workflows/start-ga.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      - name: Install Dependencies
+        run: |
+          npm -v
+          npm ci
+
+      - name: Start Release
+        env:
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+        run: |
+          node scripts/release.js ga \
```
Comment (Anemy) [inline .github/workflows/start-ga.yaml:46]: Would it work to use typescript/ts-node to run this file? Any reason we don't want to? Then we could have the file typescript.

Comment (mcasimir) [reply .github/workflows/start-ga.yaml:46]: Sounds simple enough to try, i'm not sure `scripts` is setup for that, i'll check

Comment (mcasimir) [reply .github/workflows/start-ga.yaml:46]: yeah, scripts needs to be restructured and setup for that .. i'm not particularly keen in doing it in this PR, i'd rather do it in a separate PR, and then rebase

**Review by lerouxb — APPROVED** [review]
👍

**Pull request merged**